makefiles/features_check_system.inc.mk

```diff
@@ -0,0 +1,67 @@
+# Check if all required FEATURES are provided
```

Comment:
Unrelated?

Reply:
true some thing went worng there i will remove that

Reply:
thank you

Reply:
i will remove with rebase